rental.rb

```diff
@@ -0,0 +1,16 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date)
+    @date = date
```

Comment:
- [ ] Since there is arelationship between `person and book` with rentals as intermediate class, kindly initialize these two classes in the constructor as well. 👍 
```suggestion
  def initialize(date, person, book)
    @date = date
```

Reply:
@OyePriscilla i think i have already done that at line 7 and 8